packages/turbo-workspaces/src/managers/yarn.ts

```diff
@@ -94,24 +89,39 @@ async function create(args: CreateArgs): Promise<void> {
   );
   packageJson.packageManager = `${to.name}@${to.version}`;
 
-  if (!options?.dry) {
-    fs.writeJSONSync(project.paths.packageJson, packageJson, { spaces: 2 });
-  }
+  if (hasWorkspaces) {
```

Comment:
This series of things is bothering me having seen it three times, but I don't have an alternative proposal.

`pnpm` being weird makes it probably not worth abstracting out.